src/Umbraco.Cms.Integrations.SEO.Semrush/Api/Management/Controllers/Token/RefreshAccessTokenController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Asp.Versioning;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Cms.Integrations.SEO.Semrush.Configuration;
+using Umbraco.Cms.Integrations.SEO.Semrush.Services;
+
+namespace Umbraco.Cms.Integrations.SEO.Semrush.Api.Management.Controllers.Token
+{
+    public class RefreshAccessTokenController : TokenControllerBase
+    {
+        public RefreshAccessTokenController(IOptions<SemrushSettings> options, IWebHostEnvironment webHostEnvironment, ISemrushTokenService semrushTokenService, ICacheHelper cacheHelper, TokenBuilder tokenBuilder, SemrushComposer.AuthorizationImplementationFactory authorizationImplementationFactory) : base(options, webHostEnvironment, semrushTokenService, cacheHelper, tokenBuilder, authorizationImplementationFactory)
+        {
+        }
+
+        [HttpPost("refresh")]
+        [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
```

Comment:
Similar to a comment above, might consider handling `500` as the service could return a string with `error` message.